tests/unit-tests/test_gnn.py

```diff
@@ -93,6 +96,19 @@ def check_node_prediction(model, data):
     assert_almost_equal(pred1[0:len(pred1)].numpy(), pred2[0:len(pred2)].numpy(), decimal=5)
     assert_equal(labels1.numpy(), labels2.numpy())
 
+def check_mlp_node_prediction(model, data):
```

Comment:
maybe our test should explain why we want to have this test case.

Reply:
Agree.
I put some docstring to tests in test_gnn.py

Reply:
i don't see docstring in this function.